pkg/logql/engine_test.go

```diff
@@ -1207,6 +1304,54 @@ func TestEngine_RangeQuery(t *testing.T) {
 				},
 			},
 		},
+		{
+			`bytes_over_time({app="foo"}[30s]) > bool 1`, time.Unix(60, 0), time.Unix(120, 0), 15 * time.Second, 0, logproto.FORWARD, 10,
```

Comment:
hahahaha this is so cool.

I can already see someone doing alert because he doesn't want to log too much.

/cc @slim-bean